tests/functional/u/undefined/undefined_loop_variable.py

```diff
@@ -1,9 +1,12 @@
 # pylint: disable=missing-docstring,redefined-builtin, consider-using-f-string, unnecessary-direct-lambda-call
 
+from typing import NoReturn
+
+
 def do_stuff(some_random_list):
     for var in some_random_list:
         pass
-    return var # [undefined-loop-variable]
+    return var  # [undefined-loop-variable]
```

Comment:
I let `black` reformat this file a bit. I find it really annoying when auto-formatting doesn't work on these test files 😅

Reply:
I understand, but see below for a concern about the variables checker. In general I'd prefer not to do formatting fixes on test files.